util/gdas_init/set_fixed_files.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Can you add a short prolog that explains what the script does?

Reply:
I've added it in the script as suggested.